gpkit/interactive/sensitivity_map.py

```diff
@@ -119,8 +122,8 @@ def solution(self):
             return self.model.solution
 
     def _repr_latex_(self):
-        "iPython LaTeX representation."
-        return "$$\\require{color}\n"+self.latex+"\n$$"
+        "iPython LaTeX representation"
+        return "$$\\require{color}\n" +self.latex+"\n$$"+ self.color_list
```

Comment:
Sorry if it seems confusing, I change the color list into a string at a point in the code, so that is actually a string being appended to the return